packages/components/src/components/Icon/icons.ts

```diff
@@ -115,4 +115,5 @@ export const ICONS = {
     ANCHOR: require('../../images/icons/anchor.svg'),
     PENCIL_LINE: require('../../images/icons/pencil-line.svg'),
     KEY: require('../../images/icons/key.svg'),
+    CLOSE: require('../../images/icons/close.svg'),
```

Comment:
This icon is named  **CROSS** and is already in the project.

Reply:
eb9750cc134ed0fb8ec75d435421489ad4f796fc